### DISCUSSION TITLE
Why the change from Angular to Vue?

### DISCUSSION THREAD

**Discussion by arturcarvalho** [nasa/openmct#5998] [Community Q&A] [answered]
I hope this is not too off-topic, but why did you change away from Angular?

Thanks a lot =)

**Comment by jvigliotta** [comment, marked as answer]
Hi @arturcarvalho, I wasn't here when the initial decision was made, but I believe we were on AngularJS at the time and it was nearing it's end of life. I think talks were had about going to Angular 2, but other frontend frameworks were considered and we ultimately decided Vue was a good fit. We're currently on Vue 2, but we're in the process of migrating to Vue 3.

**Reply by arturcarvalho** [reply]
Hi @jvigliotta thanks for the answer!
